src/app/components/cards/CollegeCard.tsx

```diff
@@ -35,11 +35,14 @@ export const CollegeCard = ({ college }: Props) => {
           />
         </div>
       </CardBody>
-      <CardFooter>
+      <CardFooter className="card_footer">
         <ButtonGroup>
           <Button name="apply" type={"button"}>
             Apply to this school
           </Button>
+          <Button name="information" type={"button"} outline={true}>
+            More information
+          </Button>
```

Comment:
Dan [removed this](https://github.com/coforma/swift-tech-challenge/pull/78) so just checking if we want to add it back in.

Reply:
Removed!